systemtest/driver/flow.py

```diff
@@ -28,14 +28,27 @@ def read_flow(tmp_path: Path):
             engine="python",
         )
         flow.columns = ["time", "peds", "pedIDs"]
-        flow_dict[door_id] = flow.shape[0]
+        flow_dict[door_id] = flow
 
     assert (
         len(flow_dict.keys()) > 0
     ), "Could not find flow file to expected flow"
     return flow_dict
 
 
+def read_outflow_from_inifile(inifile: Path):
+    tree = ET.parse(inifile)
+    root = tree.getroot()
+    for tc in root.iter("traffic_constraints"):
+        outflow_dict = {
+            int(door.attrib["trans_id"]): float(door.attrib["outflow"])
+            for door in tc.iter("door")
+        }
+
+    assert outflow_dict, "Could not read outflow from inifile"
+    return outflow_dict
+
+
```

Comment:
This code should be moved to `inifile.py`

Reply:
done.